src/estimator/python.rs

```diff
@@ -11,6 +11,8 @@ impl From<EstimatorError> for PyErr {
 #[macro_export]
 macro_rules! impl_estimator {
     ($eos:ty, $py_eos:ty) => {
+        /// Loss function that is applied to the residuals to 
+        /// weight to in- and outliers.
```

Comment:
slightly confusing sentence. Below it says to distinguish between in- and outliers.

Reply:
You are correct. I changed it to "[loss functions can be] applied to residuals to handle outliers" (also in the docstring for the data set).